### ISSUE TITLE
create new wallet id for new user

### ISSUE THREAD

**Issue by bonomat** [bitbetwin/bitbetwin#39]
whenever a user registers on our side a new wallet id should be generated

**Comment by holzeis** [comment]
what do we need a wallet id for? is it something unique for the btc payment?

**Comment by bonomat** [comment]
yes we do, it's his private address to which he can send his bitcoins
